<ui version="4.0" >
 <class>mediaEditForm</class>
 <widget class="QWidget" name="mediaEditForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>487</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="1" column="1" >
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item row="2" column="0" >
      <widget class="QLabel" name="label_3" >
       <property name="text" >
        <string>Pool:</string>
       </property>
       <property name="buddy" >
        <cstring>poolCombo</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="0" >
      <widget class="QLabel" name="label_5" >
       <property name="text" >
        <string>Volume Status:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0" >
      <widget class="QLabel" name="label_13" >
       <property name="text" >
        <string>Max Volume Bytes:</string>
       </property>
       <property name="buddy" >
        <cstring>slotSpin</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1" >
      <widget class="QComboBox" name="statusCombo" />
     </item>
     <item row="10" column="1" >
      <widget class="QComboBox" name="recyclePoolCombo" />
     </item>
     <item row="4" column="0" >
      <widget class="QLabel" name="label_4" >
       <property name="text" >
        <string>Slot:</string>
       </property>
       <property name="buddy" >
        <cstring>slotSpin</cstring>
       </property>
      </widget>
     </item>
     <item row="7" column="0" >
      <widget class="QLabel" name="label_12" >
       <property name="text" >
        <string>Max Volume Jobs:</string>
       </property>
       <property name="buddy" >
        <cstring>slotSpin</cstring>
       </property>
      </widget>
     </item>
     <item row="6" column="0" >
      <widget class="QLabel" name="label_7" >
       <property name="text" >
        <string>Use Duration:</string>
       </property>
       <property name="buddy" >
        <cstring>slotSpin</cstring>
       </property>
      </widget>
     </item>
     <item row="13" column="0" colspan="3" >
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="okButton" >
         <property name="text" >
          <string>OK</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancelButton" >
         <property name="text" >
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="8" column="1" >
      <widget class="QSpinBox" name="maxBytesSpin" >
       <property name="maximum" >
        <number>2147483647</number>
       </property>
      </widget>
     </item>
     <item row="5" column="0" >
      <widget class="QLabel" name="label_6" >
       <property name="text" >
        <string>Retention:</string>
       </property>
       <property name="buddy" >
        <cstring>slotSpin</cstring>
       </property>
      </widget>
     </item>
     <item row="10" column="0" >
      <widget class="QLabel" name="label_15" >
       <property name="text" >
        <string>Recycle Pool:</string>
       </property>
       <property name="buddy" >
        <cstring>slotSpin</cstring>
       </property>
      </widget>
     </item>
     <item row="12" column="0" colspan="2" >
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QCheckBox" name="enabledCheck" >
         <property name="layoutDirection" >
          <enum>Qt::RightToLeft</enum>
         </property>
         <property name="text" >
          <string>Enabled</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="9" column="1" >
      <widget class="QSpinBox" name="maxFilesSpin" >
       <property name="maximum" >
        <number>2147483647</number>
       </property>
      </widget>
     </item>
     <item row="9" column="0" >
      <widget class="QLabel" name="label_14" >
       <property name="text" >
        <string>Max Volume Files:</string>
       </property>
       <property name="buddy" >
        <cstring>slotSpin</cstring>
       </property>
      </widget>
     </item>
     <item rowspan="13" row="0" column="2" >
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="9" column="0" colspan="2" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>97</width>
           <height>21</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="1" >
        <widget class="QSpinBox" name="yearsSpin" >
         <property name="maximum" >
          <number>999</number>
         </property>
         <property name="minimum" >
          <number>-1</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0" >
        <widget class="QLabel" name="label_2" >
         <property name="text" >
          <string>Years</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0" >
        <widget class="QLabel" name="label_11" >
         <property name="text" >
          <string>Seconds</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1" >
        <widget class="QSpinBox" name="daysSpin" >
         <property name="maximum" >
          <number>30</number>
         </property>
         <property name="minimum" >
          <number>-1</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2" >
        <widget class="QRadioButton" name="useDurationRadio" >
         <property name="text" >
          <string>Use Duration</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>97</width>
           <height>31</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="5" column="0" >
        <widget class="QLabel" name="label_8" >
         <property name="text" >
          <string>Days</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1" >
        <widget class="QSpinBox" name="secondsSpin" >
         <property name="maximum" >
          <number>60</number>
         </property>
         <property name="minimum" >
          <number>-1</number>
         </property>
        </widget>
       </item>
       <item row="6" column="0" >
        <widget class="QLabel" name="label_9" >
         <property name="text" >
          <string>Hours</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" >
        <widget class="QLabel" name="label_16" >
         <property name="text" >
          <string>Months</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1" >
        <widget class="QSpinBox" name="minutesSpin" >
         <property name="maximum" >
          <number>60</number>
         </property>
         <property name="minimum" >
          <number>-1</number>
         </property>
        </widget>
       </item>
       <item row="6" column="1" >
        <widget class="QSpinBox" name="hoursSpin" >
         <property name="maximum" >
          <number>24</number>
         </property>
         <property name="minimum" >
          <number>-1</number>
         </property>
         <property name="value" >
          <number>0</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2" >
        <widget class="QRadioButton" name="retentionRadio" >
         <property name="text" >
          <string>Retention</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1" >
        <widget class="QSpinBox" name="monthsSpin" >
         <property name="maximum" >
          <number>12</number>
         </property>
         <property name="minimum" >
          <number>-1</number>
         </property>
        </widget>
       </item>
       <item row="7" column="0" >
        <widget class="QLabel" name="label_10" >
         <property name="text" >
          <string>Minutes</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="5" column="1" >
      <widget class="QSpinBox" name="retentionSpin" >
       <property name="maximum" >
        <number>2147483647</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2" >
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="volumeLabel" >
         <property name="maximumSize" >
          <size>
           <width>16777215</width>
           <height>48</height>
          </size>
         </property>
         <property name="text" >
          <string>Volume : </string>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="2" column="1" >
      <widget class="QComboBox" name="poolCombo" />
     </item>
     <item row="6" column="1" >
      <widget class="QSpinBox" name="useDurationSpin" >
       <property name="maximum" >
        <number>2147483647</number>
       </property>
      </widget>
     </item>
     <item row="11" column="0" colspan="2" >
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QCheckBox" name="recycleCheck" >
         <property name="layoutDirection" >
          <enum>Qt::RightToLeft</enum>
         </property>
         <property name="text" >
          <string>Recycle</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="0" column="0" colspan="2" >
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>71</width>
           <height>21</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label" >
         <property name="maximumSize" >
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
         <property name="text" >
          <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-size:12pt; font-weight:600;">Edit a Volume&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>81</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="4" column="1" >
      <widget class="QSpinBox" name="slotSpin" >
       <property name="maximum" >
        <number>10000</number>
       </property>
      </widget>
     </item>
     <item row="7" column="1" >
      <widget class="QSpinBox" name="maxJobsSpin" >
       <property name="maximum" >
        <number>2147483647</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
